provision/pull-images.sh

```diff
@@ -11,58 +11,64 @@ done
 # Install all of these images in VM for the CI
 if [ -z "${NAME_PREFIX}" ]; then
     for img in \
-        busybox:1.28.4 \
         docker.io/byrnedo/alpine-curl:0.1.7 \
-        docker.io/cilium/cc-grpc-demo:v2.0 \
-        docker.io/cilium/cilium:v1.0.7 \
-        docker.io/cilium/cilium:v1.1.6 \
-        docker.io/cilium/cilium:v1.2.5 \
-        docker.io/cilium/cilium:v1.3.0 \
+        docker.io/cilium/cilium-etcd-operator:v2.0.5 \
         docker.io/cilium/cilium-init:2018-10-16 \
         docker.io/cilium/connectivity-container:v1.0 \
         docker.io/cilium/demo-client:latest \
         docker.io/cilium/demo-httpd:latest \
+        docker.io/cilium/dnssec-client:v0.1 \
+        docker.io/cilium/docker-bind:v0.1 \
+        docker.io/cilium/docker-bind:v0.3 \
         docker.io/cilium/istio_pilot:1.0.2 \
+        docker.io/cilium/istio_proxy_debug:1.0.0 \
         docker.io/cilium/kafkaclient2:latest \
         docker.io/cilium/kafkaclient:latest \
         docker.io/cilium/microscope:1.1.2-ci \
+        docker.io/cilium/python-bmemcached:v0.0.1 \
         docker.io/cilium/starwars:v1.0 \
-        docker.io/coredns/coredns:1.2.2 \
-        docker.io/coredns/coredns:1.0.6 \
         docker.io/digitalwonderland/zookeeper:latest \
         docker.io/istio/citadel:1.0.2 \
-        docker.io/istio/galley:1.0.2 \
-        docker.io/istio/proxyv2:1.0.2 \
-        docker.io/istio/mixer:1.0.2 \
         docker.io/istio/examples-bookinfo-details-v1:1.6.0 \
         docker.io/istio/examples-bookinfo-productpage-v1:0.2.3 \
         docker.io/istio/examples-bookinfo-ratings-v1:1.6.0 \
         docker.io/istio/examples-bookinfo-reviews-v1:1.6.0 \
         docker.io/istio/examples-bookinfo-reviews-v2:1.6.0 \
+        docker.io/istio/galley:1.0.2 \
+        docker.io/istio/mixer:1.0.2 \
+        docker.io/istio/proxy_init:1.0.0 \
         docker.io/istio/proxy_init:1.0.2 \
+        docker.io/istio/proxyv2:1.0.2 \
         docker.io/kubernetes/guestbook:v2 \
-        docker.io/library/busybox:1.28.0 \
+        docker.io/library/busybox:1.28.0-glibc \
+        docker.io/library/busybox:1.28.4 \
         docker.io/library/httpd:2.4.34 \
+        docker.io/library/consul:1.1.0 \
         docker.io/library/redis:4.0.11 \
         docker.io/library/registry:2.6.2 \
+        docker.io/library/memcached:1.5.11 \
         docker.io/prom/prometheus:v2.3.1 \
         docker.io/prom/statsd-exporter:v0.6.0 \
+        docker.io/spotify/kafkaproxy:latest \
+        docker.io/tgraf/netperf:latest \
```

Comment:
Why are we pulling latest vs. a tagged version of this vs. the tagged version (v1.0) in the following line?